rst_files/data_statistical_packages.rst

```diff
@@ -51,7 +52,7 @@ The first is to setup columns and construct a dataframe by assigning names
     last_price = [4.2, 11.3, 12.1, missing]
     df = DataFrame(commod = commodities, price = last_price)
 
-Columns of the DataFrame can be accessed by name using a symbol or a ``.``
+Columns of the ``DataFrame`` can be accessed by name using a symbol or a ``.``
```

Comment:
is a symbol the colon thing? Have we previously defined this?

Reply:
Yeah, that's a symbol. We define it in introduction to types which comes before

Reply:
Probably good to put in an example, though. Done.